css-view-transitions-2/Overview.bs

```diff
@@ -983,6 +1081,75 @@ When capturing the old or new state for an element, perform the following steps
 	Note: This is written in a monkey-patch manner, and will be merged into the algorithm once the L1 spec graduates.
 </div>
 
+## Capturing and applying 'view-transition-group' ## {#vt-group-algorithm}
+
+<div algorithm="additional old capture steps (group)">
+When capturing the old state for an element, perform the following steps given a [=captured element=] |capturedElement|, an [=element=] |element| and a {{ViewTransition}} |transition|:
```

Comment:
Can you link to the algorithm where this needs to go eventually. IIUC it's this one: https://drafts.csswg.org/css-view-transitions-1/#capture-old-state-algorithm

Reply:
Done